jobfunnel/config/settings.yaml

```diff
@@ -22,9 +22,9 @@ search_terms:
   keywords:
     - 'Python'
 
+# Black-listed company names
 black_list:
   - 'Infox Consulting'
-  - 'Terminal'
 
 # Logging level options are: critical, error, warning, info, debug, notset
-log_level: 'error'
+log_level: 'info'
```

Comment:
I changed this because by logging WARNING level, there is no way to see what is going on. 

Recommend adding a separate configurable`log_level` for console and log file